tasks/libs/common/junit_upload_core.py

```diff
@@ -224,11 +223,13 @@ def upload_junitxmls(team_dir: Path):
         processes.append(Popen(DATADOG_CI_COMMAND + args, bufsize=-1, env=process_env, stdout=PIPE, stderr=PIPE))
 
     for process in processes:
-        _, stderr = process.communicate()
+        stdout, stderr = process.communicate()
         print(f" Uploaded {len(tuple(team_dir.iterdir()))} files for {team_dir.name}")
+        print(stdout)
```

Comment:
Is this a debug leftover or is it wanted?

Reply:
It was wanted, as it's now "cached" under the collapsible section we can print the `stdout` to have information, it's useful in case of issue